content/docs/zero/billing.md

```diff
@@ -0,0 +1,164 @@
+
+- For questions about billing, see [**How billing works.**](#how-billing-works)
+- For questions about upgrading or downgrading subscriptions, see [**Manage subscriptions**](#manage-subscriptions).
+
+## Billing definitions
+
+- **Quota**: the amount of a given entity you can define in Pomerium Zero based on your subscription type
+- **Entity**: a route, policy, certificate, service account, custom domain, API user, replica, or cluster
+
+## Organizations
+
+Pomerium Zero assigns accounts to organizations. There are two types of organizations:
+
+- **Personal Account**: assigned to you by default when you register for an account
+- **Professional**: available only if you upgrade from a **Free** plan to a **Business** plan
+
+### Plans and subscriptions
+
+Pomerium Zero provides two types of plans:
+
+- **Free**: intended for testing purposes, hobbyists, and small teams. Personal accounts subscribe to the Free plan by default.
+- **Business**: intended for large teams. Professional organizations subscribe to the Business plan.
+
+### Quotas
+
+Personal and professional organizations are allocated different quotas. Personal accounts have lower quotas, and professional organizations have much higher quotas.
+
+The table below breaks down quotas by organization type:
+
+| Entity | Personal account quota | Professional org quota |
+| --- | --- | --- |
+| Routes | 10 | 100 |
+| Policies | 5 | 100 |
+| Certificates (user uploaded) | 10 | 100 |
+| Custom domains | 1 | 5 |
+| Service accounts | 2 | 20 |
+| API users | 2 | 1,000 |
+| Clusters | 1 | 5 |
+| Replicas | 3 | 5 (per cluster) |
+
+User quotas differ also based on the organization type:
+
+| User Type   | Personal account | Professional org |
+| ----------- | ---------------- | ---------------- |
+| Admin users | 1                | 20               |
+| Users       | 10               | 1,000            |
+
+:::note
+
+See the [Pricing page](https://www.pomerium.com/pricing) for a thorough breakdown of features and quotas in Pomerium Zero.
+
+:::
+
+### Quota thresholds
+
+At some point, you may require higher quotas to accommodate your use case.
+
+If you’re using a professional account and you need to increase the quota for a given entity, [contact us](/docs/community). We will increase quotas on an as-needed basis.
+
+If you’re using a personal account, you must upgrade to increase quotas.
+
+## How billing works
+
+Pomerium charges your account monthly based on usage during a billing period. The billing period starts at the beginning of a calendar month and ends at the end of the month.
+
+Pomerium starts monitoring your usage when you upgrade and charges you for total usage at the end of the billing period.
+
+### Usage
+
+When you upgrade, you can choose a monthly or annual rate:
```

Comment:
We don't support annual.

Reply:
I removed the billing rates and kept $9 in the equation example.